.ci/cico_build_and_deploy_custom_rh-che.sh

```diff
@@ -11,28 +11,21 @@ set +o nounset
 
 /usr/sbin/setenforce 0
 
-export CHE_DOCKER_BASE_IMAGE=eclipse/che-server:nightly-centos
+export RH_CHE_AUTOMATION_SERVER_DEPLOYMENT_URL=https://rhche-che6-automated.dev.rdu2c.fabric8.io/
+export BASEDIR=$(pwd)
 export ORIGIN_CLIENTS_URL=http://mirror.centos.org/centos/7/paas/x86_64/openshift-origin/origin-clients-3.7.1-2.el7.x86_64.rpm
-export OPENSHIFT_RDU2C_URL=https://dev.rdu2c.fabric8.io:8443/
+export OSD_RDU2C_URL=https://dev.rdu2c.fabric8.io:8443/
 export OC_VERSION=3.9.19
-
-function tagAndPushDocker() {
-  export RH_CHE_AUTOMATION_BUILD_TAG=rh-che-automated-build-$(git rev-parse --short HEAD)
-  echo "che-docker-base-image:${CHE_DOCKER_BASE_IMAGE}"
-  docker pull ${CHE_DOCKER_BASE_IMAGE}
-  docker tag ${CHE_DOCKER_BASE_IMAGE} eclipse/che-server:local
-  rm -rf ./dockerfiles/che-fabric8/eclipse-che
-  mkdir ./dockerfiles/che-fabric8/eclipse-che
-  cp -r ./assembly/assembly-main/target/eclipse-che-fabric8-1.0.0-SNAPSHOT/eclipse-che-fabric8-1.0.0-SNAPSHOT/* ./dockerfiles/che-fabric8/eclipse-che/
-  bash ./dockerfiles/che-fabric8/build.sh || exit 1
-  docker rmi eclipse/che-server:local
-  docker rmi ${CHE_DOCKER_BASE_IMAGE}
-  docker tag eclipse/che-server:nightly rhcheautomation/che-server:${RH_CHE_AUTOMATION_BUILD_TAG}
-  docker rmi eclipse/che-server:nightly
-  set +x
-  docker login -u ${RH_CHE_AUTOMATION_DOCKERHUB_USERNAME} -p ${RH_CHE_AUTOMATION_DOCKERHUB_PASSWORD}
-  set -x
-  docker push rhcheautomation/che-server:${RH_CHE_AUTOMATION_BUILD_TAG}
+export TARGET="rh-aut"
+export RH_CHE_DOCKER_IMAGE_VERSION=nightly-rhcheautomation
```

Comment:
Could you please point me in what registry this image would be pushed during PR check ?

Reply:
https://github.com/ibuziuk/rh-che/pull/1/files/93cece52a25a8dc63039736938d013321c19c5b2#diff-e73e617c2211a7e2ce254344615e4fc2R124
`push.registry.devshift.net/osio-prod/rhchestage/rh-che-automation:<upstream_tag>-rhcheautomation-<che_commit_short_hash>`

Reply:
I think that one is an unused variable. Will check and clean up